lxc/completion.go

```diff
@@ -189,7 +189,7 @@ func (g *cmdGlobal) cmpRemotes() ([]string, cobra.ShellCompDirective) {
 	results := []string{}
 
 	for remoteName, rc := range g.conf.Remotes {
-		if rc.Protocol != "incus" && rc.Protocol != "" {
+		if rc.Protocol != "lxd" && rc.Protocol != "" {
```

Comment:
Could you add this to the previous commit please? We don't have to take the original cherry-pick as-is, as long as we mark the attribution and the licence of the original commit.